src/image.rs

```diff
@@ -53,7 +53,7 @@ pub enum ImageQuality {
 /// Owned shareable image resource.
 #[derive(Clone, PartialEq, Debug)]
 #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
-pub struct Image {
+pub struct ImageData {
```

Comment:
There are arguments for e.g. `ImageBitmap` or even `Bitmap` here. But I think `ImageData` is probably better than those.